HACKING.md

```diff
@@ -489,6 +489,29 @@ oc annotate secret/additional-root-ca-cert \
 The description can be customized. All other fields must be
 exactly as is.
 
+### [OPTIONAL] Create splunk secret
+
+The Splunk secret is actually a configuration stored in Bitwarden as
+secret, here is an example:
+
+```
+unclassified:
+  splunkJenkinsInstallation:
+    enabled: true
+    globalPipelineFilter: true
+    host: ***
+    port: 8088
+    splunkAppUrl: ***
+    token: ***
+    useSSL: true
+```
+To create it use:
+```
+oc create secret generic splunk-casc-cfg \
+    --from-literal=splunk.yaml \
+    --from-file=splunk.yaml
```

Comment:
This looks off. I don't think we want the `--from-literal` switch? The `--from-file` switch should do what we want (create a key named `splunk.yaml` with the contents of the file).